app/src/vanilla/java/cm/aptoide/pt/view/AppCoinsInfoNavigator.java

```diff
@@ -0,0 +1,42 @@
+package cm.aptoide.pt.view;
+
+import android.os.Bundle;
+import cm.aptoide.pt.R;
+import cm.aptoide.pt.app.view.NewAppViewFragment;
+import cm.aptoide.pt.link.CustomTabsHelper;
+import cm.aptoide.pt.navigator.ActivityNavigator;
+import cm.aptoide.pt.navigator.FragmentNavigator;
+
+import static cm.aptoide.pt.app.view.AppCoinsInfoFragment.APPCWALLETPACKAGENAME;
+
+/**
+ * Created by D01 on 02/08/2018.
+ */
+
+public class AppCoinsInfoNavigator {
+
+  private final ActivityNavigator activityNavigator;
+  private final FragmentNavigator fragmentNavigator;
+
+  public AppCoinsInfoNavigator(ActivityNavigator activityNavigator,
+      FragmentNavigator fragmentNavigator) {
+
+    this.activityNavigator = activityNavigator;
+    this.fragmentNavigator = fragmentNavigator;
+  }
+
+  public void navigateToCoinbaseLink() {
+    CustomTabsHelper.getInstance()
+        .openInChromeCustomTab(activityNavigator.getActivity()
+            .getString(R.string.coinbase_url), activityNavigator.getActivity());
+  }
+
+  public void navigateToAppCoinsBDSWallet() {
+    NewAppViewFragment appViewFragment = new NewAppViewFragment();
+    Bundle bundle = new Bundle();
+    bundle.putString(NewAppViewFragment.BundleKeys.PACKAGE_NAME.name(), APPCWALLETPACKAGENAME);
+    bundle.putString(NewAppViewFragment.BundleKeys.STORE_NAME.name(), "bds-store");
```

Comment:
should probably be a constant like appc wallet package name.

Reply:
I didn't created it because for now it's only being used here in 1 place, unlike the package name